models/record.go

```diff
@@ -446,6 +446,8 @@ func (rc *RecordConfig) ToRR() dns.RR {
 		rr.(*dns.TLSA).Certificate = rc.GetTargetField()
 	case dns.TypeTXT:
 		rr.(*dns.TXT).Txt = rc.TxtStrings
+	case dns.TypeDHCID:
+		rr.(*dns.DHCID).Digest = rc.GetTargetField()
```

Comment:
Please keep this list alphabetized. (and if anything else is out of place, also move it)